src/coreclr/jit/namedintrinsiclist.h

```diff
@@ -118,6 +119,9 @@ enum NamedIntrinsic : unsigned short
     NI_System_Runtime_CompilerServices_RuntimeHelpers_IsKnownConstant,
     NI_System_Runtime_CompilerServices_RuntimeHelpers_IsReferenceOrContainsReferences,
     NI_System_Runtime_CompilerServices_RuntimeHelpers_GetMethodTable,
+    NI_System_Runtime_CompilerServices_RuntimeHelpers_Await,
+    NI_System_Runtime_CompilerServices_RuntimeHelpers_AsyncSuspend,
+    NI_System_Runtime_CompilerServices_RuntimeHelpers_get_RuntimeAsyncViaJitGeneratedStateMachines,
```

Comment:
This can be deleted. It is from the times when we had another non-JIT implementation of runtime async.

Reply:
Right :-)